index.js

```diff
@@ -96,6 +103,11 @@ class LiveStrategyExecution extends EventEmitter {
       candle.symbol = symbol
       candle.tf = tf
 
+      if (candle.mts > lastUpdate) {
+        this.priceFeed.update(new BigNumber(candle.close))
```

Comment:
Better to read as the saved `candlePrice` in strategyState, although the default value is close.
```
const { candlePrice } = this.strategyState
this.priceFeed.update(new BigNumber(candle[candlePrice]))
```